src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -44,12 +45,26 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
   ArtworkSidebarCommercialContainerProps,
   ArtworkSidebarCommercialContainerState
 > {
-  state = {
-    isCommittingCreateOrderMutation: false,
-    isErrorModalOpen: false,
-    selectedEditionId:
-      this.props.artwork.edition_sets[0] &&
-      this.props.artwork.edition_sets[0].__id,
+  sortedEditionSets() {
+    const editions = this.props.artwork.edition_sets as any
+
+    return sortBy(editions, edition => edition.sale_message)
```

Comment:
How come we're sorting this? `sale_message` is text and doesn't immediately feel like it should be sortable? Since `edition_sets` is an array and is thus ordered, I think we should preserve that order?

Reply:
I added sorting due to this acceptance criteria in the ticket:

> Display the price range from lowest to highest edition prices above shipping details and ‘Buy now’ button per spec

https://artsyproduct.atlassian.net/projects/DISCO/issues/DISCO-514

Agreed that the `sale_message` is a bit odd to sort on, but it seems like the leading dollar sign followed by a numeric value ends up sorting in a way that matches the desired behavior without too much complexity. Do you think that breaking out available from unavailable editions and sorting directly on `price` would be a better solution?

Reply:
Interesting, I wasn't aware/glossed over that acceptance criteria.

I still think that it should be consistent. Partners enter edition sets in a certain order, and I would think we should just display them in that order? If it's desired that edition sets be sorted by price, then perhaps it might be better to do that on the server (Gravity), and sort by an _actual_ price, vs the proxy string-based sorting.

I'll bring it up in Slack.

Reply:
@dblandin seems like we do want to sort this low to high.

I'll add a numerical `price_cents` field for the edition sets and update MP to return edition sets sorted by that field if you do `edition_sets(sort: PRICE_ASC)` or something. Then we can just update the fragment here and we can remove all the sorting from this file.

Reply:
Sounds great!